tdrs-frontend/.env

```
@@ -45,3 +45,5 @@ INLINE_RUNTIME_CHUNK=false
 
 # Ignore CRA package version check until version is updated to 5.0.0
 SKIP_PREFLIGHT_CHECK=true
+
+REACT_APP_LOGIN_XMS_URL=https://int.xms.hhs.gov/ABaaS/idp
```

After trying to use this with @raftmsohani and @jorgegonzalez on an XMS branch, we have decided on the below edits.  Can this be renamed and made into a simple boolean?

```suggestion
LOGIN_XMS_USED=True
```

---

This should be lowercase in JavaScript @abottoms-coder